addon/templates/components/has-gravatar.hbs

```diff
@@ -1,6 +1,6 @@
 {{#unless checking}}
   {{yield (hash
-    has=has
+    has=hasGravatar
```

Comment:
Notice: you can see that we are still yielding `has` in order to keep the established API.